docs/manual/java/guide/production/ProductionOverview.md

```diff
@@ -14,33 +12,35 @@ The production environment determines the methods for packaging your services, m
 
     * Otherwise, you can implement the interface yourself to integrate with a service registry of your choosing (such as [Consul](https://www.consul.io/), [ZooKeeper](https://zookeeper.apache.org/), or [etcd](https://coreos.com/etcd/)) or start with an open-source example implementation such as [`lagom-service-locator-consul`](https://github.com/jboner/lagom-service-locator-consul) or [`lagom-service-locator-zookeeper`](https://github.com/jboner/lagom-service-locator-zookeeper).
 
-* Services that require an [Akka Cluster](https://doc.akka.io/docs/akka/current/cluster-usage.html) (which includes any that use the Lagom [[Persistence|PersistentEntity]] or [[Publish-Subscribe|PubSub]] APIs) must have a strategy for forming a cluster or joining an existing cluster on startup.
+* Lagom services that require an [Akka Cluster](https://doc.akka.io/docs/akka/current/cluster-usage.html) (which includes any that use the Lagom [[Persistence|PersistentEntity]] or [[Publish-Subscribe|PubSub]] APIs) must have a strategy for forming a cluster or joining an existing cluster on startup.
 
     * If you don't use a service orchestration platform and can determine the addresses of some of your nodes in advance of deploying them, Akka Cluster can be configured manually by listing the addresses of [seed nodes](https://doc.akka.io/docs/akka/current/cluster-usage.html#joining-to-seed-nodes) in the service configuration.
 
-    * Lightbend Orchestration includes open-source support for [automatic Akka Cluster formation](https://developer.lightbend.com/docs/lightbend-orchestration/current/features/akka-cluster-bootstrap.html) on Kubernetes or DC/OS.
+    * When you don't specify a no static list of `seed-nodes`, Lagom will enable the [Akka Cluster Bootstrap](https://developer.lightbend.com/docs/akka-management/current/bootstrap.html) extension for integration with other service discovery infrastructure. Using Akka Cluster Bootstrap requires setting up a `Service Discovery` method (for example [Service Discovery for Kubernetes](https://developer.lightbend.com/docs/akka-management/current/discovery/kubernetes.html)). See the [[Lagom Cluster|Cluster]] documentation for more information.
+
+* The Akka Cluster Bootstrap feture introduced above requires enabling [Akka management HTTP](https://developer.lightbend.com/docs/akka-management/current/). That is an Akka HTTP server running on [port 8558](https://github.com/akka/akka-management/blob/v1.0.0-RC3/management/src/main/resources/reference.conf#L17) by default(that port is configurable) and shared by multiple features. Lagom will start the Akka management HTTP automatically anytime a feature that requires it is enabled. See also the [[Lagom Cluster|Cluster#Akka-Management]] documentation.
```

Comment:
Second sentence a bit hard to parse, maybe:
The Akka HTTP server is shared by multiple features. It's port number runs on [port 8558](https://github.com/akka/akka-management/blob/v1.0.0-RC3/management/src/main/resources/reference.conf#L17) by default, but is configurable.